graylog2-server/src/main/java/org/graylog2/indexer/searches/SearchesConfigBuilder.java

```diff
@@ -74,6 +74,6 @@ public SearchesConfigBuilder setSorting(Sorting sorting) {
     }
 
     public SearchesConfig build() {
-        return new SearchesConfig(query, filter, fields, range, limit, offset, sorting);
+        return null;
```

Comment:
This will lead to strange errors if the `SearchesConfigBuilder` class is being used somewhere. Can't we just remove the class?